pkg/debug/checkups/services.go

```diff
@@ -0,0 +1,224 @@
+package checkups
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"runtime"
+
+	"golang.org/x/sys/windows/svc"
+	"golang.org/x/sys/windows/svc/mgr"
+)
+
+const kolideSvcName = "LauncherKolideK2Svc"
+
+type servicesCheckup struct {
+	data         map[string]any
+	serviceState svc.State
+}
+
+func (s *servicesCheckup) Name() string {
+	return "Service Report"
+}
```

Comment:
```suggestion
func (s *servicesCheckup) Name() string {
	if runtime.GOOS != "windows" {
		return ""
	}
	return "Service Report"
}
```

Reply:
Though given that svc only exist on windows, we're going to need to build constrain this anyhow :|